docs/interaction-analytics-api.md

```diff
@@ -215,150 +221,150 @@ print(response.text)
+| tone               | List[SpeakerId-Value]   | List of SpeakerId-Value Segments.   | The SpeakerId-Value Segment is defined below.   |
+| talk_to_listen     | List[SpeakerId-Value]   | List of SpeakerId-Value Segments.   | The SpeakerId-Value Segment is defined below.   |
+| intent_timings     | List[Intent-Timings]    | List of Intent-Timings Segments.    | The Intent-Timings Segment is defined below.    |
+| key_phrase_timings | List[KeyPhrase-Timings] | List of KeyPhrase-Timings Segments. | The KeyPhrase-Timings Segment is defined below. |
+| calleq             | CallEQ-Object           | Aggregate level metrics.            | The CallEQ-Object is defined below.             |
 
 
 #### Diarized-Segment
 
-| Parameter     | Type   | Description                                           | Notes |
-| ------------- | ------ | ----------------------------------------------------- | ----- |
-| speaker_id    | String | The speaker id for the corresponding audio segment    |       |
-| start         | Number | Start time of the audio segment in seconds            |       |
-| end           | Number | End time of the audio segment in seconds              |       |
-| text          | String | The transcription output corresponding to the segment |       |
-| confidence    | Number | The confidence score for the transcribed segment      |       |
-| emotion       | String | Emotion corresponding to the segment                  | Will be computed only if specified via metrics |
-| emotion_score | Number | The confidence score for the emotion                  | Will be computed only if emotion is computed |
-| intent        | String | Intent corresponding to the segment: Current intent list: budget, authority, timing, action, need  | Will be computed only if specified via metrics |
+| Parameter     | Type   | Description                                            | Notes |
+| ------------- | ------ | ------------------------------------------------------ | ----- |
+| speaker_id    | String | The speaker id for the corresponding audio segment.    |       |
+| start         | Number | Start time of the audio segment in seconds.            |       |
+| end           | Number | End time of the audio segment in seconds.              |       |
+| text          | String | The transcription output corresponding to the segment. |       |
+| confidence    | Number | The confidence score for the transcribed segment.      |       |
+| emotion       | String | Emotion corresponding to the segment.                  | Will be computed only if specified via metrics. |
+| emotion_score | Number | The confidence score for the emotion.                  | Will be computed only if emotion is computed.   |
+| intent        | String | Intent corresponding to the segment: Current intent list: `budget`, `authority`, `timing`, `action`, `need`. | Will be computed only if specified via metrics. |
```

Comment:
can you add `action`, `decision` to the list

Reply:
done